include/flatbuffers/util.h

```diff
@@ -682,6 +682,26 @@ inline std::string BufferToHexText(const void *buffer, size_t buffer_size,
   return text;
 }
 
+/// @brief Get a pointer to the the file_identifier section of the buffer.
```

Comment:
Why are these in util.h and some other methods such as the getroot ones in base.h ? Maybe better to stick them all together into buffer.h, where we collect stuff that works on entire buffers?

Also, util.h is meant to be optional, i.e. not needed for core flatbuffer operation, most dealing with files and text related issues. The idea being that if you are just doing pure binary (de)serialization, you need flatbuffers.h but not util.h

Reply:
Yeah, i was a little haphazard on where i placed some of the free functions. But your idea is good about putting it into buffer.h Thanks for the clarification.